flexmeasures/ui/static/openapi-specs.json

```diff
@@ -3410,7 +3411,7 @@
   },
   "info": {
     "title": "FlexMeasures",
-    "version": "0.28.1.dev56"
```

Comment:
This version number seems unexpectedly outdated, given that we're currently working on 0.30.0.

Reply:
Hm, not an issue in this PR - that version comes from here:

`from flexmeasures import __version__ as fm_version`

Looks alright to me, should be based on the tag, but maybe something needs to be fixed.

I'll go ahead and merge this one, however.